packages/training/book/src/training-pipeline/gpu-profiles.md

```
@@ -0,0 +1,236 @@
+vllm_cmd = [
+    "python", "-m", "vllm.entrypoints.openai.api_server",
+    "--model", profile["model"],
+    "--tensor-parallel-size", str(profile.get("tensor_parallel", 1)),
+    "--gpu-memory-utilization", str(profile["vllm_gpu_memory"]),
+]
+```
+
+## Creating Custom Profiles
+
+1. Copy an existing profile:
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)